### DISCUSSION TITLE
i still don't know how to host a server-side rendering example

### DISCUSSION THREAD

**Discussion by queekao** [vitejs/vite-plugin-react#163] [Q&A]
like in the dist folder we have client and server how to host them?? so confuse if i host server use node it will show the import react is not existing and if i host on nginx it doesnt show anything. help me plz

**Comment by ArnaudBarre** [comment]
Hey missed that discussion,

The ssr example are quite advanced example to show how you can start doing some SSR with Vite. If you want something more mature with documentation, look at https://vite-plugin-ssr.com/
